screw-your-neighbor-server-openapi.json

```diff
@@ -2860,35 +4402,7 @@
         }
       },
       "EntityModelCard": {
-        "type": "object",
-        "properties": {
-          "cardRank": {
-            "type": "string",
-            "enum": [
-              "SIX",
-              "SEVEN",
-              "EIGHT",
-              "NINE",
-              "TEN",
-              "JACK",
-              "QUEEN",
-              "KING",
-              "ACE"
-            ]
-          },
-          "cardSuit": {
-            "type": "string",
-            "enum": [
-              "CLUB",
-              "DIAMOND",
-              "HEART",
-              "SPADE"
-            ]
-          },
-          "_links": {
-            "$ref": "#/components/schemas/Links"
-          }
-        }
+        "$ref": "#/components/schemas/Card"
```

Comment:
just reference the Card that we don't have to duplicate the information about the properties.